server/models/TestPlanRun.js

```diff
@@ -10,9 +10,9 @@ module.exports = function(sequelize, DataTypes) {
                 primaryKey: true,
                 autoIncrement: true
             },
-            isManuallyTested: { type: DataTypes.BOOLEAN, defaultValue: false },
-            tester: { type: DataTypes.INTEGER, allowNull: true },
-            testPlanReport: { type: DataTypes.INTEGER }
+            testerUserId: { type: DataTypes.INTEGER, allowNull: true },
+            testPlanReportId: { type: DataTypes.INTEGER },
+            results: { type: DataTypes.JSONB }
```

Comment:
Previous comment in the migration script on the `results` attribute type applies here as well.